keyboards/kradoindustries/kousa/info.json

```
@@ -0,0 +1,145 @@
+{
+    "keyboard_name": "Kousa-TKL", 
+    "manufacturer": "Krado Industries",
+    "url": "https://www.kradoindustries.com/", 
+    "maintainer": "Krado Industries", 
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    "usb": {
+        "vid": "0x12D0",
+        "pid": "0x4B4F",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "mousekey": true,
+        "extrakey": true,
+        "console": false,
+        "command": false,
+        "nkro": true
+    },
+
+    "rgblight": {
+        "led_count": 18,
+        "pin": "GP4",
+        "hue_steps": 10,
+        "saturation_steps": 17,
+        "brightness_steps": 17,
+        "animations": {
+            "all": true,
```

Could you explicitly define which animations are enabled? 
```suggestion
            "alternating": true,
            "breathing": true,
            "christmas": true,
            "knight": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "rgb_test": true,
            "snake": true,
            "static_gradient": true,
            "twinkle": true
```